internal/pkg/api/handleCheckin_test.go

```diff
@@ -115,6 +115,24 @@ func TestConvertActionData(t *testing.T) {
 		aType:  UNENROLL,
 		expect: Action_Data{},
 		hasErr: false,
+	}, {
+		name:   "migrate action - nil input fails",
+		aType:  MIGRATE,
+		raw:    nil,
+		expect: Action_Data{},
+		hasErr: true,
+	}, {
+		name:   "migrate action - missing required field",
```

Comment:
I can tell that this PR isn't related with this behaviour but why actions that are missing `required` fields are not expected to return an err? 🤔